src/memory/readGameMemory.ahk

```diff
@@ -79,6 +80,19 @@ readGameMemory(d2rprocess, settings, playerOffset, ByRef gameMemoryData) {
         ReadMobs(d2rprocess, startingOffset, mobs)
     }
 
+    ; missiles
+    missiles:=[]
+    ; PlayerMissiles
+    if (settings["showPlayerMissiles"]){
+        PlayerMissiles:=ReadUnits(d2rprocess, "0x21EA4F0", "Playermissiles")
```

Comment:
I'd probably look at having a different way to specify this offset, probably just need (UnitHashTableOffset+ 1024*whatever)

Reply:
Indeed! I'm learning a lot and so far the table hasn't moved! I'm still figuring out a lot but just as the automap class you noticed, in my readunits i have a switch case for different units. once I figure out the offest I will apply it but TBH ive never really messed with memory and offsets... like I get it but I've never really cared enough to do it. Now I do :p